pkg/cmd/opts/install.go

```diff
@@ -780,7 +780,28 @@ func (o *CommonOptions) InstallGcloud() error {
 	if runtime.GOOS != "darwin" || o.NoBrew {
 		return errors.New("please install missing gcloud sdk - see https://cloud.google.com/sdk/downloads#interactive")
 	}
-	return o.RunCommand("brew", "cask", "install", "google-cloud-sdk")
+	latestVersion := "279.0.0"
```

Comment:
We should not hard code version for external binaries here. That's what the version stream is for. See for example how it is done for the Helm install - https://github.com/jenkins-x/jx/blob/ea98d9eb2f0b2b0be06e0126e384e050f303f9cc/pkg/cmd/opts/install.go#L355

For this to work you will need to create a pull request in this repo first https://github.com/jenkins-x/jenkins-x-versions/tree/master/packages where you add a _ gcloud.yml_ specifying the version.

Reply:
@hferentschik Thanks for the feedback.  I already have made a PR to that repository https://github.com/jenkins-x/jenkins-x-versions/pull/811